lib/cluster_ht_item.rb

```diff
@@ -32,4 +32,17 @@ def move(new_cluster)
     end
   end
 
+  # Removes an HTItem
+  def delete
+    Cluster.where("ht_items.item_id": @ht_item.item_id).each do |c|
+      c.ht_items.delete_if {|h| h.item_id == @ht_item.item_id }
+      Reclusterer.new(c).recluster
+    end
+  end
+
+  # Deletes an HTItem, then re-adds it
+  def update
+    delete
```

Comment:
This is definitely the simplest implementation and good for now. I think we will probably want to add an optimization for the usual case where an update doesn't change OCNs, but I think we can wait for that until we have a process that handles the updates and we have a sense of how many things get updated and how long it takes to process on a daily basis.

Reply:
When I ran this I was seeing ~10  records/second. This implies somewhere in the range of an hour to handle an average days worth of updates depending upon a slew of other variables. Manageable, but definitely not ideal.

I changed it to update_attributes for an HTItem, and only delete/recluster when an OCN changed. Running a days worth of updates twice (add, then update) took ~2 minutes as opposed to the hour long naive approach.